modules/ROOT/partials/toolbar-button-ids/advtable-toolbar-buttons.adoc

```diff
@@ -2,5 +2,8 @@
 | Toolbar button identifier | Description
 
 | `advtablerownumbering`
-| Adds or removes (toggles) a row numbering column on the selected table. include::partial$misc/requires_5_9v.adoc[]
+| Adds or removes (toggles) a row numbering column on the selected table.
+
+include::partial$misc/requires_5_9v.adoc[]
```

Comment:
```suggestion
| Adds or removes (toggles) a row numbering column on the selected table.
include::partial$misc/requires_5_9v.adoc[]
```
Missed this as well, but since this is part of this line we should keep it together.